tests/ha/fencing.pm

```diff
@@ -32,8 +32,8 @@ sub run {
         # Fence the node
         assert_script_run 'crm -F node fence ' . get_node_to_join;
 
-        # Wait a little to be sure that fence command is on his way
-        sleep 60;
+        # We also need to reset the console on this node
+        reset_consoles;
```

Comment:
another question - in this case you first wait for barrier and then reset console , and in another case you doing this other way around. Is this random or you really need different order in each case ?

In case it is random you can probably simplify this peace to something like :
```
reset_consoles;
barrier_wait("BEFORE_FENCING_$cluster_name");
 # Fence the node
assert_script_run 'crm -F node fence ' . get_node_to_join if is_node(2);
```

Reply:
I tried one time to simply like this but I had strange behaviour. Other thing is that I want to have reset_consoles executed *after* the last action on all nodes, in your example it's not the case for 2nd node, but that's more cosmetic.

But I could try again to simplify this code.